.github/workflows/testing.yml

```diff
@@ -124,6 +124,7 @@ jobs:
             ovn: ovn
           - extra-toggles: dual-stack
           - extra-toggles: ovn
+          - extra-toggles: ovn-ic
```

Comment:
We dont run OVN-IC in 0.15 right, so why this change?

Reply:
Good point, I’ll fix https://github.com/submariner-io/shipyard/pull/1395 to drop that.